src/utils/crypto.ts

```diff
@@ -0,0 +1,42 @@
+import crypto from "crypto";
+
+// encryptionKey must be 256 bits (32 characters)
+const IV_LENGTH: number = 16; // For AES, this is always 16
+
+export function encrypt(encryptionKey: string): (text: string) => string {
```

Comment:
This looks really good, but I have one suggestion. Specifying the encryption key as a string is easy, but means that a 32-character string isn't really giving you 256 bits of key, because it's highly likely that those characters are going to come from a narrow alphabet; e.g. if it's a 32-character string of base64 characters, that's six bits per character, not eight, so it's really a 192-bit key. So I'd suggest allowing the encryption key to be specified as a byte array of 32 bytes, and providing some helper functions to convert from either base64 or hex string into a byte array.

Reply:
I've changed the implementation quite a bit, any type and length of string should now be supported.